apps/storybook-react/.storybook/main.ts

```diff
@@ -31,6 +31,7 @@ const config: StorybookConfig = {
     name: getAbsolutePath('@storybook/react-vite'),
     options: {},
   },
+  staticDirs: [{ from: '../images', to: '/images' }],
```

Comment:
Sets a list of directories of [static files](https://storybook.js.org/docs/configure/integration/images-and-assets#serving-static-files-via-storybook-configuration) to be loaded by Storybook.